draft-tigress-requirements.md

```diff
@@ -141,8 +141,6 @@ General terms:
 # Requirements
 
 - (Req-AnyPlatform) Solution SHOULD be able to communicate with any mobile device of any operating system and allow easy implementation of server-side components without requiring a specific Cloud stack.
-- (Req-NontechnicalUX) Solution SHALL enable secure credential transfer for non technical users.
```

Comment:
I think this requirement is important because it precludes sending a raw data blob that needs to be copy and pasted to be redeemed. Without this requirement there is not as big of a need for a new protocol because people could send key sign requests over an already secure messaging channel.

I have a feeling this type of requirement is common the IETF ART area. Is there a common way to phase this that would be acceptable? "The protocol should be usable"

Reply:
are there any examples of these types of requirements in the ART area that we can use as a pattern?

Reply:
I do not have an example, if you have not seem similar requirements in the past then I am ok to remove both UX requirements.